sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ModelBridgeInternal.java

```diff
@@ -72,8 +72,8 @@ public static CosmosAsyncDatabaseResponse createCosmosAsyncDatabaseResponse(Reso
         return new CosmosAsyncItemResponse<>(response, classType);
     }
 
-    public static CosmosAsyncItemResponse createCosmosAsyncItemResponseWithObjectType(ResourceResponse<Document> response, Class classType) {
-        return new CosmosAsyncItemResponse(response, classType);
+    public static CosmosAsyncItemResponse<Object> createCosmosAsyncItemResponseWithObjectType(ResourceResponse<Document> response) {
+        return new CosmosAsyncItemResponse<>(response, Object.class);
```

Comment:
I think this should have T and Class<T> instead of Object

Reply:
@moderakh - the method `createCosmosAsyncItemResponseWithObjectType` is called from only one place with second parameter as `Object.class`. [here](https://github.com/Azure/azure-sdk-for-java/blob/2c32ea66cefeec207d9c88d5bff79961fc6e5cef/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java#L524). Hence I thought of simplifying it since the method name itself convey that its Object Type. Do you see any issues?

Reply:
@moderakh  - yes this particular API is specifically for Object type -> as this is being called by deleteItem(String id) -> which doesn't take any class type information. So this is for generic object type.